src/SprykerEco/Yves/AmazonPay/AmazonPayDependencyProvider.php

```diff
@@ -26,6 +27,7 @@ class AmazonPayDependencyProvider extends AbstractBundleDependencyProvider
     public const CLIENT_CUSTOMER = 'customer client';
     public const CLIENT_CALCULATION = 'calculation client';
     public const CLIENT_GLOSSARY_STORAGE = 'glossary storage client';
+    public const CLIENT_MESSENGER = 'messenger client';
```

Comment:
I see that it was done to be the same as the rest of the constants, but value should be the same as a constant name